analyzer/analyzer.spicy

```diff
@@ -2,6 +2,9 @@
 
 module DHCP;
 
+# Global variable containing the DHCP message type of the currently parsed message
+public global dhcp_type: MessageType;
+
```

Comment:
Could you please remove all changes which are already in #5?

Reply:
Removed.